src/course_bot/presentation.clj

```diff
@@ -57,7 +57,7 @@
           group (-> data (get id) :presentation (get pres-key) :group)
           lessons (-> data :presentation (get pres-key) (get group))]
       (->> lessons
-           (filter (fn [[_k v]] (-> v :stud-ids empty? not)))
+           (filter (fn [[_k v]] (-> v :stud-ids seq)))
```

Comment:
About this suggestion. Revert your fix and suppress that warning. From my point of view -- it is very bad practice.

Here and in other places.

Reply:
Suppressed such warning in config